### DISCUSSION TITLE
Where can I find documentation for cross.toml?

### DISCUSSION THREAD

**Discussion by wpd** [cross-rs/cross#532] [Q&A] [answered]
Specifically, I am wondering if it is possible to specify the target in the cross.toml (or even the cargo.toml) file.

Thank you.

--wpd

**Comment by jwupf** [comment]
I'm interested in this question as well. How can I setup a default target, so that cross run uses a non-native environment.

**Comment by Emilgardis** [comment, marked as answer]
~Currently, there is no documentation.~ I've created https://github.com/cross-rs/cross/issues/664 for tracking this need.

see https://github.com/cross-rs/cross/blob/main/docs/cross_toml.md

**Reply by Emilgardis** [reply]
We've added a [cross_toml.md](https://github.com/cross-rs/cross/blob/main/docs/cross_toml.md) file for this now

**Reply by jwupf** [reply]
Thanks

**Reply by JayKickliter** [reply]
I appreciate the existence of `cross_toml.md`, but it doesn't really explain why/how those thins are meant to be used.

**Reply by Emilgardis** [reply]
@JayKickliter what are the things you don't understand? feel free to make a new discussion or reach out on our matrix room https://matrix.to/#/#cross-rs:matrix.org

**Reply by JayKickliter** [reply]
Thanks! I started #742
